e2e/page-objects/bugReporting.po.ts

```diff
@@ -5,4 +5,32 @@ export class BugReportingPage {
   async getPhaseDescription() {
     return element(by.css('app-root')).element(by.id('phase-descriptor')).getText();
   }
+
+  async accessNewBugReportingPage() {
+    return element(by.className('create-new-bug-report-button')).click();
+  }
+
+  /**
+   * Identifies a Bug-Report based on the provided parameters.
+   * @param title Title of Bug-Report.
+   * @param severityLabel Severity assigned to Bug-Report.
+   * @param bugTypeLabel Bug-Report's Type.
+   * @returns true if a unique Bug-Report is present, false otherwise.
+   */
+  async isBugReportPresent({title, severityLabel, bugTypeLabel}: {title: string, severityLabel?: string, bugTypeLabel?: string}) {
```

Comment:
Introduced the parameters here in a de-structured format due to the optional parameters. This would allow for cleaner parameter passing when invoking the function, since we'd be able to exactly define which parameters are instantiated and which are left undefined like seen in  `bugReportingPhase.e2e-spec.ts`